source/lambda/testing_requirements.txt

```diff
@@ -1,7 +1,7 @@
 botocore>=1.12.99
 boto3>=1.9.99
```

Comment:
Technically boto3 and botocore are held back in lambda runtimes, I try to match, but it probably doesn't make a difference: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html